ge03_team01/models/product.py

```diff
@@ -0,0 +1,16 @@
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    name = fields.Char("Name", index="trigram", translate=True, compute="_compute_name", store=True, readonly=False)
+
+    @api.depends("make", "model", "year")
+    def _compute_name(self):
+        for record in self:
+            if record.detailed_type is not None and record.detailed_type == "motorcycle" and False not in (
```

Comment:
You do not need the 'is not None'. You could change the condition to something like: 

`if all([template.year, template.make, template.model]) and template.detailed_type == "motorcycle"`